README.md

```diff
@@ -1,2 +1,95 @@
-# __template__
-ReasonML / BuckleScript bindings for ORIGINAL_PACKAGE_NAME
+**How to use this template**
+
+- Put your bindings in `src/ReactNativeSomething` & rename accordingly or use
+  `bsconfig.json` `"namespace"` field (more on this below),
+- Update all occurences of
+
+  - `@reason-react-native/__template__`
+  - `https://github.com/reason-react-native/__template__`
+  - `@react-native-community/something`
```

Comment:
And more importantly `@reason-react-native/something`.

Reply:
I have modified to have `@reason-react-native/__template__` (bindings) & `@react-native-community/something` (original package) only

Reply:
The build status still refers to reason-react-native/reason-react-native though.

Reply:
fixed